### PR TITLE
[MLP 205]  Create page for listing existing roles

### PR DESCRIPTION
**Pull request by alcabelu**
# Background
Build a Page for listing the roles, consuming a WCF host in http://localhost:10160/Service.svc/Jsonp/post with method post an in body json request {"requests":[{"__type":"RoleReadAllRequest:#P202.Training.WCF.RequestsAndResponses"}]}, build with durandal and knockout for binding 

# Changes done
- Add folder Role in webApp
- Add file roleList.html in folder Role
- Add file roleList.js in folder Role
- Add file roleService.js in folder service
- Add Code in routes.js for Role
- Add functionality For Page for RoleList

# Pending to be done
Nothing

# Notes
For execution npm start,
In web explorer open localhost:8080
And select RoleList

# Appovers required
@martinkeimel
@gmsoft
@darkainsoul

### CODE REVIEW

**Review by martinkeimel — APPROVED** [review]
Looks great :1st_place_medal:

File: webapp/app/viewModels/role/roleList.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                <th>Id</th>
+                <th>Rol Description</th>
+                <th>Created By</th>
+                <th>Rol Action</th>
+            </tr>
+        </thead>
+        <tbody data-bind="foreach : roles">
+            <tr>
+                <td>                  
+
```
Comment (GustavoAlMena) [inline webapp/app/viewModels/role/roleList.html]: Ctl+L ;)

File: webapp/app/viewModels/role/roleList.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      return $.parseHTML(view)[0];
+    },
+    roles: ko.observableArray([])   
+    };
+    
+
+    vm.activate = function () {
+
+        return roleService.roleReadAll().then(function (response) {
+            console.log(response.ProcessJsonRequestsPostResult[0].Roles);
```
Comment (GustavoAlMena) [inline webapp/app/viewModels/role/roleList.js]: When you finish you debug task remove this line

**Review by GustavoAlMena — APPROVED** [review]
CR Approved!

File: webapp/app/viewModels/role/roleList.js (hunk 2 above)
Comment (alcabelu) [reply webapp/app/viewModels/role/roleList.js]: i remove the console.log

File: webapp/app/viewModels/role/roleList.html (hunk 1 above)
Comment (alcabelu) [reply webapp/app/viewModels/role/roleList.html]: i made this chane

**Pull request merged**